addon/components/rdf-input-fields/simple-value-input-field.js

```diff
@@ -27,9 +42,32 @@ export default class SimpleValueInputFieldComponent extends InputFieldComponent
   }
 
   updateValue(value) {
-    updateSimpleFormValue(this.storeOptions, value, this.nodeValue);
+    let literalOrValue;
+
+    if (rdflib.isLiteral(value)) {
+      literalOrValue = value;
+    } else {
+      literalOrValue = this.nodeValue?.copy();
```

Comment:
This ensures we use the same language as before, we could create a new literal to do this as well but this seemed better?